packages/core/src/command/patient-import/patient-import-local.ts

```diff
@@ -49,35 +67,53 @@ export class PatientImportHandlerLocal implements PatientImportHandler {
     rerunPdOnNewDemographics,
     dryrun,
   }: ProcessPatientImportRequest): Promise<void> {
-    await createJobRecord({
-      cxId,
-      jobId,
-      data: { jobStartedAt },
-      s3BucketName,
-    });
-    const patients = await validateAndParsePatientImportCsvFromS3({
-      cxId,
-      jobId,
-      s3BucketName,
-    });
-    if (dryrun) return;
-    const processPatientCreateRequests: ProcessPatientCreateRequest[] = patients.map(patient => {
-      const patientPayload = createPatientPayload(patient);
-      return {
+    const { log } = out(`PatientImport proces patient import - cxId ${cxId} jobId ${jobId}`);
```

Comment:
nit: can't just use the function name as prefix to `out()`?

the more we add the harder it is to grok through logs

Reply:
Done.